packages/web/src/components/spa/HackAUBG/FAQSection/FAQSection.jsx

```diff
@@ -0,0 +1,224 @@
+                            registering, because if the registered teams become
+                            more than 12, your team will be waitlisted until the
+                            very last week before the competition.`}></FAQModule>
+                <FAQModule
+                    title={`Gather a team and register`}
+                    text={`Gather your dream team and register in the form
+                            below! Don’t forget to create a diverse team, as the
+                            best teams usually incorporate programmers, business
+                            planners, and designers. The teams should be between
+                            4 and 6 people. However, if you have less, don’t
+                            worry! Organizers from the Hub will add you to the
+                            best suited team for you. Don’t procrastinate
+                            registering, because if the registered teams become
+                            more than 12, your team will be waitlisted until the
+                            very last week before the competition.`}
+                    border={'faq-hide-border'}></FAQModule>
+                <FAQModule
+                    title={`Gather a team and register`}
+                    text={`Gather your dream team and register in the form
+                            below! Don’t forget to create a diverse team, as the
+                            best teams usually incorporate programmers, business
+                            planners, and designers. The teams should be between
+                            4 and 6 people. However, if you have less, don’t
+                            worry! Organizers from the Hub will add you to the
+                            best suited team for you. Don’t procrastinate
+                            registering, because if the registered teams become
+                            more than 12, your team will be waitlisted until the
+                            very last week before the competition.`}></FAQModule>
+                <FAQModule
+                    title={`Gather a team and register`}
+                    text={`Gather your dream team and register in the form
+                            below! Don’t forget to create a diverse team, as the
+                            best teams usually incorporate programmers, business
+                            planners, and designers. The teams should be between
+                            4 and 6 people. However, if you have less, don’t
+                            worry! Organizers from the Hub will add you to the
+                            best suited team for you. Don’t procrastinate
+                            registering, because if the registered teams become
+                            more than 12, your team will be waitlisted until the
+                            very last week before the competition.`}></FAQModule>
+                <FAQModule
```

Comment:
Check out how Isak did this: 

https://github.com/AUBGTheHUB/monolith/pull/632/files#diff-3e5555b192b3b5b5d134642d8c183c33b19700b641b0eacc504da286b54eb706